apps/demos/.testcaferc.json

```diff
@@ -4,7 +4,8 @@
   ],
   "screenshots": {
     "path": "./",
-    "pathPattern": "${TEST}.png"
+    "pathPattern": "${TEST}.png",
+    "textDiffTreshold": 0.1
```

Comment:
For what scenarios is this setting necessary? Does it affect all etalons and does it not work without it?